bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/MenuSheetView.java

```diff
@@ -253,6 +258,27 @@ public void setColumnWidthDp(int columnWidthDp) {
     }
 
     /**
+     * Sets a custom layout for displaying a list item when of type {@link MenuType#LIST}
+     * Call this before {@link #onAttachedToWindow()} gets called if you want to see the custom view immediately,
+     * otherwise default layout will be used.
+     * @param listItemRes needs to have an {@link ImageView} with id set to R.id.icon and {@link TextView} with id set to R.id.label
+     */
+    public void setListItemRes(int listItemRes) {
```

Comment:
I've added an `@LayoutRes` annotation so that it's expected to be a layout resource reference